src/framework/components/collision/trigger.js

```diff
@@ -41,13 +41,14 @@ Object.assign(pc, function () {
                 shape.calculateLocalInertia(mass, localInertia);
 
                 var pos = entity.getPosition();
+                var offsetPos = pos + data.offset;
```

Comment:
There is no operator overloading in JavaScript. I'm assuming this line doesn't work as you expect? Instead, I'd probably just change the line further down to say `origin.setValue(pos.x + offset.x, pos.y + offset.y, pos.z + offset.z);`.